glide.lock

```diff
@@ -536,10 +596,12 @@ imports:
   - kubernetes/typed/authorization/v1beta1/fake
   - kubernetes/typed/autoscaling/v1
   - kubernetes/typed/autoscaling/v1/fake
-  - kubernetes/typed/autoscaling/v2alpha1
-  - kubernetes/typed/autoscaling/v2alpha1/fake
+  - kubernetes/typed/autoscaling/v2beta1
```

Comment:
I'm just curious why we need something like this there? Is this a pruned set of dependencies or is there a reason why we need to import so many of these packages.
For example, if we use v2beta1 for something, why do we also need to add a dependency to v1.

Reply:
I think this is because of how clientsets work - they pull code for all supported versions.

Reply:
No idea, just ran `glide up -v`